CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
     Changes that have landed in master but are not yet released.
     Click to see more.
   </summary>
+
+  * BREAKING: return date object from date/datetime widgets if no format set ([@erquhart](https://github.com/erquhart) in [#1296](https://github.com/netlify/netlify-cms/pull/1296))
```

Comment:
Maybe it would be better to say "Possibly Breaking", since we're not doing a major release.

Reply:
Yeah, agreed